docs/using.rst

```diff
@@ -410,6 +410,13 @@ Renewing certificates
    days). Make sure you renew the certificates at least once in 3
    months.
 
+.. seealso:: Many of the certbot clients obtained through a
+   distribution come with automatic renewal out of the box,
+   such as Debian and Ubuntu versions installed through `apt`,
+   CentOS/RHEL 7 through EPEL, etc.  See `Automated Renewals`_
+   for more details, or consult your distribution's
+   documentation.
+
```

Comment:
I have a vague preference to move this to the part of the section where we talk about cron renewals.
As such I'd move it to be after the paragraph that ends on line 524